Client/TabManagement/TabManagerImplementation.swift

```diff
@@ -3,7 +3,117 @@
 // file, You can obtain one at http://mozilla.org/MPL/2.0/
 
 import Foundation
+import TabDataStore
+import Storage
+import Common
+
+// Hard coding a feature flag while the feature is in development
+// This will be replaced with a nimbus flag when the feature is ready
+// FXIOS-5885
+var useNewTabDataStore = false
```

Comment:
nit, I put the hard coded flag for coordinators in AppConstants, I think it would be nice to have hard coded flags for features under development in one place. It can be somewhere else than app constants. What do you think ?